src/components/Typography/types.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Types for typography component
+ * @param {number} size - Size of the text (pixels)
+ * @param {string} label - Text content of the compopnent
+ */
+
+export interface ITypographyProps extends Partial<HTMLHeadingElement> {
+  size: number;
+  label: string;
```

Comment:
please rename the file name to be `typography.types.d.ts`

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/20/commits/acfb9043d165fe9a9b30dc621c1b84e1e4d5bf39